.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+## Add new chain
+
+> Template to provide information about the new chain. Only add extra information in the bottom section. Ensure that the contracts are deployed on the chain, if not deploy with [safe-contracts](https://github.com/safe-global/safe-contracts). This entire paragraph can be deleted.
+
+Please fill the following form:
+
+Provide the Chain ID (Only 1 chain id per PR).
+- Chain_ID: 
+
+Provide RPC URL for the chain (should be able to query atleast 10 requests per minute for automatic PR check).
+- RPC_URL: 
+
```

Comment:
I wish there was a way to get this programmatically from Chainlist... My concern is that someone can put in an RPC for an "unofficial" node that lies and gives false sense of security with the :heavy_check_mark: in CI.

Is there a way to programatically get this somehow? I tried snooping on Chainlist but didn't find anything.

I'm OK with it as-is for now, but something to potentially improve in the future.

Reply:
There could be something we can write in bash to extract the RPC from the list on the Chainlist website, but it will be highly dependent on their website structure. We could have a list of things which we can improve on the script as a future iteration, and this could be one of them.

Reply:
Added to TODO 👍🏾